src/main/java/htsjdk/samtools/sra/SRAAccession.java

```diff
@@ -107,7 +130,20 @@ public static boolean isValid(String acc) {
 
         if (!looksLikeSRA) return false;
 
-        return isSupported() && NGS.isValid(acc);
+        ExceptionInInitializerError initError = checkIfInitialized();
+        if (initError != null) {
+            if (noLibraryDownload && initError instanceof LibraryLoadError) {
+                throw new LinkageError(
+                        "Failed to load SRA native libraries and auto-download is disabled. " +
+                        "Please set property samjdk.sra_libraries_download=true to enable auto-download of native libraries",
```

Comment:
you might want to be explicit about the -D as you'll find that many users will not know how to do this unless you are very explicit...